src/views/shelter/single.handlebars

```diff
@@ -13,23 +13,47 @@
     <dd>{{shelter.killShelter}}</dd>
   </dl>
 
-  {{#if shelter.reviews.length}}
-    <h3>Reviews</h3>
+  {{#if shelter.pets.length}}
+    <h3>Pets</h3>
     <ul>
-      {{#each shelter.reviews}}
-        <li>{{this}}</li>
+      {{#each shelter.pets}}
+        <li><a href="/petdetails/{{this.id}}">{{this.name}}</a></li>
       {{/each}}
     </ul>
-  {{else}}
-    <p>No reviews yet</p>
   {{/if}}
+</div>
 
-  {{#if shelter.pets.length}}
-    <h3>Pets</h3>
+  {{#if shelter.reviews.length}}
+    <h3>Reviews</h3>
     <ul>
-      {{#each shelter.pets}}
-        <li><a href="/petdetails/{{this.id}}">{{this.name}}</a></li>
+      {{#each shelter.reviews}}
+        <li>{{this.reviewerName}} - Review: "{{this.review}}" Rating: {{this.rating}}</li>
       {{/each}}
     </ul>
+  {{else}}
+    <p>No reviews yet</p>
   {{/if}}
-</div>
\ No newline at end of file
+<p>
+    {{error}}
+</p>
+<div id="diverror"></div>
+<form method="post" id="addReview">
+    <label id="nameLabel">
+        Reviewer Name
+        <input type="text" id="name" name="name" placeholder="Reviewer Name"/>
+    </label>
+    <label id="reviewLabel">
+        Review
+        <input type="text" id="review" name="review" placeholder="Review"/>
+    </label>
+    <label id="ratingLabel">
+        Review
```

Comment:
Please change from Review to Rating on line 50

Reply:
Just fixed it, thanks!